horovod/spark/lightning/remote.py

```diff
@@ -111,7 +111,7 @@ def train(serialized_model):
         _checkpoint_callback = None
         require_checkpoint = False
 
-        with remote_store.get_local_output_dir() as run_output_dir:
+        with remote_store.get_local_run_dir() as run_output_dir:
```

Comment:
Why the existing `get_local_output_dir()` is not working here? 
It is an existing local tmp path.

Reply:
`get_local_output_dir()` creates a random temporary directory which is then copied to run path, but when loading checkpoints it looks for a specific directory named 'checkpoint' which is not created, hence this change.

Reply:
got it. thx!